tests/integration/defs/llmapi/test_llm_api_connector.py

```
@@ -542,6 +542,46 @@ def test_connector_priorities_default(enforce_single_worker,
     assert request.priorities is None
 
 
+@pytest.mark.threadleak(enabled=False)
+def test_connector_load_error_terminates_request(enforce_single_worker,
+                                                  model_with_connector):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)